tests/dynamic_threshold_ndvi.html

```diff
@@ -0,0 +1,123 @@
+      #map {
+        bottom: 0;
+        left: 0;
+        position: absolute;
+        right: 0;
+        top: 0;
+      }
+
+      .input-wrapper {
+        position: fixed;
+        right: 16px;
+        top: 16px;
+        z-index: 1000;
+        padding: 8px;
+        background: white;
+        border: 1px solid black;
+        border-radius: 8px;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="map">
+    </div>
+    <div class="input-wrapper">
+      <input id="threshold" min="0" max="1" type="range" step="0.1" />
+    </div>
+    <script src="https://unpkg.com/browse/whatwg-fetch@3.2.0/dist/fetch.umd.js"></script>
+    <script src="https://unpkg.com/chroma-js"></script>
+    <script src="https://unpkg.com/leaflet@1.0.3/dist/leaflet.js"></script>
+    <script src="https://unpkg.com/georaster"></script>
+    <script src="https://unpkg.com/proj4"></script>
+    <script src="../georaster-layer-for-leaflet.min.js"></script>
+    <script>
+      // initalize leaflet map
+      var map = L.map("map").setView([0, 0], 5).setMaxZoom(25);
+      let layer;
+
+      // add OpenStreetMap basemap
+      L.tileLayer("http://{s}.tile.osm.org/{z}/{x}/{y}.png", {
+        attribution:
+          '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors',
+      }).addTo(map);
+
+      /*
+        NDVI is (Near-Infrared - Red) / (Near-Infrared + Red)
+
+        "In Landsat 8, NDVI = (Band 5 – Band 4) / (Band 5 + Band 4)." (USGS)
+      */
+        function getColorFromValues(values, min, max) {
+          const [ RED, NIR ] = values;
+          const dividend = NIR - RED;
+          const divisor = NIR + RED;
+          let result;
+          if (dividend === 0 && divisor === 0) {
+            // probably no reading here
+            return 'rgba(0,0,0,0)';
+          }
+          if (dividend === 0 || divisor === 0) {
+            result = 0;
+          } else {
+            result = dividend === 0 ? 0 : dividend / divisor;
+          }
+          if (result <= 0.1) return 'blue';
+          if (result >= 0.8) return 'black';
+          return chroma.scale(["#B31C08", "#E6E60B", "#1FB308"]).domain([min, max])(result).hex();
+        };
+
+      const baseURL = "https://landsat-pds.s3.amazonaws.com/c1/L8/024/030/LC08_L1TP_024030_20180723_20180731_01_T1/LC08_L1TP_024030_20180723_20180731_01_T1_B.TIF";
+      const band4URL = baseURL.replace('B.TIF', 'B4.TIF');
+      const band5URL = baseURL.replace('B.TIF', 'B5.TIF');
+      const scale = chroma.scale([
+        '#C7BB95',
+        '#FEFEE1',
+        '#6E9F62',
+        '#032816',
+        'black'
+      ]).domain([
+        0,
+        .2,
+        .4,
+        .6,
+        .8
+      ]);
+      Promise.all([
+        parseGeoraster(band4URL),
+        parseGeoraster(band5URL)
+      ]).then(function (georasters) {
+        layer = new GeoRasterLayer({
+          georasters,
+          pixelValuesToColorFn: function(values) {
+            return getColorFromValues(values, 0, 1)
```

Comment:
I think this is missing a semi-colon at the end.

Reply:
Nice one thanks. Would you be happy with me adding a `.prettierrc` to the project? I had an issue where my IDE started auto formatting the files using my global file instead of the prettier defaults. It would normally use whatever was in the root folder. Happy to do this in a separate PR.

Reply:
As an example, this is the config we use: 

```json
{
  "semi": true,
  "singleQuote": true,
  "tabWidth": 2,
  "trailingComma": "all",
  "useTabs": false,
  "arrowParens": "avoid",
  "bracketSpacing": true,
  "htmlWhitespaceSensitivity": "css",
  "insertPragma": false,
  "jsxBracketSameLine": false,
  "jsxSingleQuote": false,
  "printWidth": 80,
  "proseWrap": "always",
  "quoteProps": "as-needed",
  "requirePragma": false,
  "vueIndentScriptAndStyle": false
}
```

Reply:
Good catch.  I didn't realize this was an issue.  I've created a ticket for the task here: https://github.com/GeoTIFF/georaster-layer-for-leaflet/issues/56.  I can do it once we get this PR into the main trunk.  (I'll have to do other small things like updating the package.json, etc. before re-publishing).